wire-runtime/build.gradle

```diff
@@ -53,6 +53,10 @@ task japicmp(type: JapicmpTask, dependsOn: 'jar') {
   packageExcludes = [
       'com.squareup.wire.internal',
   ]
+  classExcludes = [
+      // package protected -> public    
+      'com.squareup.wire.RuntimeMessageAdapter',      
```

Comment:
Gets accessed from `wire-gson-support` so has to be public

Reply:
Is this still needed?

Reply:
Thought it would be since the class is missing. Japicmp doesn't complain though, so I removed it.